www/wiki/js/mod/admin/js/index.js

```diff
@@ -373,6 +385,187 @@ define([
 			};
 		}
 		
+		
+		//商品管理/goodsManager
+		$scope.getGoodmnagerList = function(){
+			$scope.selectMenuItem = "goodsManager";
+			$scope.getGoods();
+			$scope.listGoodsCount();
+		}
+		
+		//商品管理判断是否为添加/修改/查看详情
+		
+		$scope.clickGoodsToggle = function (params,item) {
+			$scope.goodsVar = params;
+			/*if(params == 1){
+				$scope.getOneGoodsInfo(item);
+			}*/
+			if(params == 2){
+				$scope.goodsParams = {};
+				$scope.goodsMan = [];
+				$scope.goodsParams.is_on_sale = 1;
+			}
+			if(params == 3){
+				$scope.getOneGoodsInfo(item);
+			}
+		};
+
+		$scope.ceshidianji = function(item){
+			goodsFactory.itemSet(item.additional_field)
+			$uibModal.open({
+				"animation"       :true,
+				"ariaLabeledBy"   :"title",
+				"ariaDescribedBy" :"body",
+				"template"        :`

```

Comment:
Some browsers do not support template strings, please check caniuse.com before using ES6 feature.